extensions/scarb-cairo-run/src/main.rs

```diff
@@ -111,19 +116,41 @@ fn main() -> Result<()> {
         )
         .context("failed to run the function")?;
 
+    let profiling_info = if args.run_profiler {
+        let profiling_info_processor =
+            ProfilingInfoProcessor::new(sierra_program.program, Default::default());
```

Comment:
Not sure what should be the second argument here, found [this](https://github.com/starkware-libs/cairo/blob/main/crates/bin/cairo-run/src/main.rs#L91) in compiler repo